recipe/meta.yaml

```diff
@@ -1,54 +1,59 @@
 {% set name = "pantable" %}
-{% set version = "0.12.4" %}
+{% set version = "0.13.3" %}
+
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 9ad940e6088285241a642f812f5cab83aa98491682884876c1205b2103afff01
+  sha256: 600b4687355b14ae5e3d53fe568f0192622ee21e37202ef577f63b1893e112cb
 
 build:
   noarch: python
   number: 0
   entry_points:
     - pantable = pantable.cli.pantable:main
     - pantable2csv = pantable.cli.pantable2csv:main
-  script: "{{ PYTHON }} -m pip install . -vv"
+    - pantable2csvx = pantable.cli.pantable2csvx:main
+  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
-    - python >=3.6,<4
     - pip
+    - python >=3.7
+    - poetry >=1,<2
   run:
-    - python >=3.6,<4
-    - panflute >=1.8.2,<2
+    - backports.cached-property >=1,<2  # [py<38]
```

Comment:
Our usual solution is to just remove the selector and install the package for all python versions. `backports.cached-property` will install fine on Python >=3.8. It's a lot better (build-wise) than moving this from `noarch`. You may want to leave a comment noting that the dependency can be removed when we stop supporting Python 3.7.